store/db.go

```diff
@@ -33,6 +34,9 @@ type DB struct {
 	dbdir string
 	lock  *lock.FileLock
 	sqldb *sql.DB
+	// In fact, the mutex here is not necessary, as we already
+	// use flocks. This is just for making race detector happy.
```

Comment:
@jonboulle Why? The first thing is Open() is to acquire a flock, and the last thing in Close() is to close the flock. All the code between these two lines are in the critical section.